lib/Data/Database.php

```diff
@@ -598,39 +630,39 @@ private static function _getTableQuery($type)
     {
         switch ($type) {
             case 'ibm':
-                $sql = 'SELECT tabname FROM SYSCAT.TABLES ';
+                $sql = 'SELECT "tabname" FROM "SYSCAT"."TABLES"';
                 break;
             case 'informix':
-                $sql = 'SELECT tabname FROM systables ';
+                $sql = 'SELECT "tabname" FROM "systables"';
                 break;
             case 'mssql':
-                $sql = 'SELECT name FROM sysobjects '
-                     . "WHERE type = 'U' ORDER BY name";
+                $sql = 'SELECT "name" FROM "sysobjects" '
+                     . 'WHERE "type" = \'U\' ORDER BY "name"';
                 break;
             case 'mysql':
                 $sql = 'SHOW TABLES';
                 break;
             case 'oci':
-                $sql = 'SELECT table_name FROM all_tables';
+                $sql = 'SELECT "table_name" FROM "all_tables"';
```

Comment:
```suggestion
                $sql = 'SELECT table_name FROM all_tables';
```

Throws `ORA-00942: table or view does not exist` otherwise

Reply:
added